### PR TITLE
mirroring: make the docs about secrets less vague

### PR DESCRIPTION
**Pull request by petr-muller**
Also, document how to add these secrets to Bitwarden for DPTP.

/cc @stevekuznetsov @droslean @bbguimaraes @hongkailiu

### CODE REVIEW

File: core-services/quay-mirroring/README.md [hunk 1]
```diff
@@ -21,7 +21,17 @@ when put to a ConfigMap.
...
 You should not need to modify anything in the job besides the items marked as
 `FIXME`, where you just need to fill in the name of your image set (it should
-be the same as the name of the subdirectory here). The only exception to this is
-the name of the secret that will allow the job to push your images to Quay. You
-need to talk to DPTP about storing this secret in DPTP vault from where it would
-be synced to the cluster.
+be the same as the name of the subdirectory here). To push images to Quay, the
+job needs credentials. These credentials need to be available in the cluster as
+a secret. The secrets are stored in DPTP Bitwarden vault and then synced to the
+cluster: talk to DPTP about storing the credentials in Bitwarden.
```
Comment (stevekuznetsov) [inline core-services/quay-mirroring/README.md]: add a slack channel link here?

File: core-services/quay-mirroring/README.md [hunk 2]
```diff
@@ -21,7 +21,17 @@ when put to a ConfigMap.
...
+
+### For DPTP: How to create new push credential secrets
```
Comment (stevekuznetsov) [inline core-services/quay-mirroring/README.md]: isn't this in our secret management doc?

Comment (petr-muller) [reply core-services/quay-mirroring/README.md]: Oh. We have a secret management doc! I'll extend the section there a bit and link it from here. Thanks for the pointer.

**Comment by petr-muller** [comment]
Addressed.

**Pull request merged**